### ISSUE TITLE
Web.config error: duplicate ‘mimeMap’

### ISSUE THREAD

**Issue by lfurzewaddock** [ridercz/AutoACME#9]
To resolve the following IIS error;

> Cannot add duplicate collection entry of type ‘mimeMap’ with unique key attribute ‘fileExtension’ set to ‘.’

I needed to add the following line to the generated web.config

`<remove fileExtension="." `/>

Perhaps its worth adding this by default?

```
<configuration>
  <system.webServer>
    <staticContent>
      <!--
        ACME server requires that the verification file is served either without
        Content-Type header or with "text/json". This setting ensures that.
      -->
      <remove fileExtension="." />
      <mimeMap fileExtension="." mimeType="text/json" />
    </staticContent>
  </system.webServer>
</configuration>
```

Great work by the way and thanks for the clear step by step instructions!

**Comment by ridercz** [comment]
Good idea,

**Comment by ridercz** [comment]
Fixed in 1.5.2
